src/modules/apiCall.js

```diff
@@ -0,0 +1,17 @@
+const dataJson = [''];
+const apiCall = async (homePageOption) => {
+  await fetch(`https://api.tvmaze.com/search/shows?q=${homePageOption[0]}`)
+    .then((data) => data.json())
+    .then((data) => {
+      dataJson.length = 0;
+      dataJson.push(data);
+    });
+
+  for (let index = 0; index < dataJson[0].length; index += 1) {
```

Comment:
- [ ] Please make sure that all tests are checked. 
![image](https://user-images.githubusercontent.com/74744787/211096723-9b0dd736-4273-486c-8a9a-58ec085971a5.png)

Reply:
All the tests are passing now.

Reply:
![ss](https://user-images.githubusercontent.com/43726580/211140588-e60af8c4-b117-43ee-a545-5ae398647b1c.jpg)